docs/src/data/hid.js

```diff
@@ -5184,7 +5184,7 @@ export default [
     os: {
       windows: null,
       linux: true,
-      android: null,
+      android: false,
```

Comment:
```suggestion
      android: true,
```
Tested and found to be working. If I can't commit this suggestion I'll open a new PR.